### DISCUSSION TITLE
Plot curves using matplotlib?

### DISCUSSION THREAD

**Discussion by talhaanwarch** [Lightning-AI/pytorch-lightning#8665] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Can we plot loss and metrics using matplotlib without using tensorboard?

**Comment by tchaton** [comment]
Dear @talhaanwarch,

Yes you can. You would have to implement a `MatplotlibLogger`.

```py
from pytorch_lightning.loggers.base import LightningLoggerBase


class MatplotlibLogger(LightningLoggerBase):

    ...


Trainer(logger=MatplotlibLogger())
```
